### PR TITLE
UI development

### PR DESCRIPTION
**Pull request by ktpa**
Now we share all of our issues through the branches <3

### CODE REVIEW

**Review by gsastraw — APPROVED** [review]
As a UI developer, I approve

File: client/app/src/main/java/group01/smartcar/client/DrivingScreen.java [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+import androidx.appcompat.app.ActionBar;
+import androidx.appcompat.app.AppCompatActivity;
+
+import static group01.smartcar.client.Status.ACTIVE;
+
+// 78 to 112 adapted from https://developer.android.com/training/system-ui/immersive .
+
+public class DrivingScreen extends AppCompatActivity implements JoystickView.JoystickListener {
+    CarControl car;
```
Comment (dekmm) [inline client/app/src/main/java/group01/smartcar/client/DrivingScreen.java]: Make private

File: client/app/src/main/java/group01/smartcar/client/MainActivity.java [hunk 2]
```diff
@@ -1,66 +1,135 @@
...
-    CarControl car;
-    protected ImageView cameraView;
-    protected TextView speedometer;
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseUser;
+
+public class MainActivity extends AppCompatActivity {
+
+    private FirebaseAuth firebaseAuth;
+    EditText emailTextView, passwordTextView;
```
Comment (dekmm) [inline client/app/src/main/java/group01/smartcar/client/MainActivity.java]: Make private

File: client/app/src/main/java/group01/smartcar/client/MainActivity.java [hunk 3]
```diff
@@ -1,66 +1,135 @@
...
+    private VideoView videoBackground;
+    MediaPlayer mediaPlayer;
```
Comment (dekmm) [inline client/app/src/main/java/group01/smartcar/client/MainActivity.java]: Make private

**Comment by ktpa** [comment]
@gsastraw why do you review this we worked on it together brudda

File: client/build.gradle [hunk 4]
```diff
@@ -8,6 +8,7 @@ buildscript {
     dependencies {
         classpath "com.android.tools.build:gradle:4.1.3"
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+        classpath 'com.google.gms:google-services:4.3.3'
```
Comment (dekmm) [inline client/build.gradle]: Move to app/build.gradle

**Pull request merged**